run.py

```diff
@@ -0,0 +1,36 @@
+
```

Comment:
Top level documentation highlighting what this module accomplishes as a whole can be added. 

eg.
"""
Enables the command line execution of multiple modules within src/
This module combines the argparsing of each module within src/ and enables the execution of the corresponding scripts
so that all module imports can be absolute with respect to the main project directory.
To understand different arguments, run `python run.py --help`
"""